library/src/main/java/com/alexfu/countdownview/CountDownView.java

```diff
@@ -103,6 +111,10 @@ public void stop() {
         timer.cancel();
     }
 
+    public void addOnCountListener(OnCountListener listener){
```

Comment:
This should be `setOnCountListener` not `addOnCountListener`. Having `add` implies that you can have more than one listener. This isn't the case here. Also, I think I would prefer it to say `setListener` instead.

Reply:
Cool